docs/en/latest/plugins/workflow.md

```diff
@@ -0,0 +1,153 @@
+---
+title: workflow
+keywords:
+  - APISIX
+  - Plugin
+  - workflow
+  - traffic control
+description: This document describes the Apache APISIX workflow Plugin, you can use it to control traffic.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Description
+
+The `workflow` plugin is used to introduce [lua-resty-expr](https://github.com/api7/lua-resty-expr#operator-list) to provide complex traffic control features.
+
+## Attributes
+
+| Name                         | Type          | Required | Default | Valid values | Description                                                  |
+| ---------------------------- | ------------- | -------- | ------- | ------------ | ------------------------------------------------------------ |
+| rules.case                   | array[array]  | True     |         |              | List of variables to match for filtering requests for conditional traffic split. It is in the format `{variable operator value}`. For example, `{"arg_name", "==", "json"}`. The variables here are consistent with NGINX internal variables. For details on supported operators, you can refer to [lua-resty-expr](https://github.com/api7/lua-resty-expr#operator-list). |
+| rules.actions                | array[object] | True     |         |              | The action to be performed when the case matches successfully. Currently, only one element is supported in actions. The first child element of the actions' only element can be `return` or `limit-count`. |
+| rules.actions[1].return      | string        | False    |         |              | Return directly to the client.                               |
```

Comment:
Could we move each action to a separate table? As the number of actions may grow fastly.

Reply:
updated